src/main/java/me/william278/huskhomes2/config/Settings.java

```diff
@@ -442,15 +454,15 @@ public int getPublicHomesPerPage() {
     public int getWarpsPerPage() {
         return warpsPerPage;
     }
-
+    @Nullable
```

Comment:
Tbh in the future I'd like to refactor the settings and it'd be better to use `Optional<Boolean>` here, but this works and is fine.